tools/static-assets/skel-typescript/package.json

```diff
@@ -16,6 +16,7 @@
   "devDependencies": {
     "@types/meteor": "^1.4.87",
     "@types/mocha": "^8.2.3",
+    "@types/node": "^18.13.0",
```

Comment:
Technically we're still at v14, but some features are polyfilled... Just a heads up.